filters/PoissonFilter.cpp

```diff
@@ -317,6 +317,14 @@ PointViewSet PoissonFilter::run(PointViewPtr view)
             transform[i++] = xform.coords[r][c];
 
     TransformationFilter().doFilter(*outView, transform);
+
+    // Rerun normals as they may be expected.
+    {
+        NormalFilter f;
```

Comment:
Must we always do this even if we are not writing normals for output according to the `m_writeNormals` option that was set?